packages/moleculer-db-adapter-mongoose/src/index.js

```diff
@@ -443,6 +446,19 @@ class MongooseDbAdapter {
 			return id.toString();
 		return id;
 	}
+
+
+	isPromise(p) {
```

Comment:
Please use the `isPromise` method from Moleculer Utils, instead of a new implementation.